.circleci/config.yml

```diff
@@ -53,10 +53,22 @@ jobs:
       - run:
           name: Run unit tests
           command: pipenv run -- make test
+  schema_tests:
+      docker:
+        - image: 'circleci/golang'
+      steps:
+        - checkout
+        - run:
+            name: Install panthelrog binary
+            command: curl -sSO https://panther-community-us-east-1.s3.amazonaws.com/v1.17.0-RC/tools/dev/pantherlog-linux-amd64 && mv pantherlog-linux-amd64 pantherlog && chmod +x pantherlog
```

Comment:
Will this URL need to be updated occasionally? Is there/do we want a "latest" tag or something for this?

Reply:
> Is there/do we want a "latest" tag or something for this?

I can add a task for the infra team to look into this.
It should be a part of the `mage publish` task.
An alternative would be to publish a Docker image for each version of the tool.
Until then, manually editing a parameter should do it.

> You can also use a parameter for better visibility and easier version changes

One thing I also considered would be to also have a SHA256 of the binary in a parameter and check the downloaded blob against it.
Will be adding the parameter